src/JsonApi/Request/Request.php

```diff
@@ -105,7 +105,7 @@ public function validateQueryParams()
     protected function isValidMediaTypeHeader($headerName)
     {
         $header = $this->getHeaderLine($headerName);
-        return (strpos($header, "application/vnd.api+json") === false || $header === "application/vnd.api+json");
+        return strpos($header, "application/vnd.api+json") !== false;
```

Comment:
I had to fix this method since it was returning true for any kind of header. After this change, it returns TRUE only if the header line is `application/vnd.api+json`. If this is how it should behave than my change is correct.